htdocs/install/mysql/tables/llx_facturedet.sql

```diff
@@ -65,8 +65,14 @@ create table llx_facturedet
   
   situation_percent real DEFAULT 100, 								-- % progression of lines invoicing
   fk_prev_id        integer, 										-- id of the line in the previous situation
+  situation_percent_cumulated real DEFAULT 100,                     -- Invoice situation percent cumulated
```

Comment:
Same remark than for invoice.
On invlice line we have a field    
fk_prev_id
It is the link to the previous invoice line.  total cumulated is sum of all lines.
getCumulatedAmountForPreviousSituationLine()  on invoice line